ui/src/context/Application.jsx

```diff
@@ -4,7 +4,7 @@ import { makeAsyncIterableFromNotifier as iterateNotifier } from '@agoric/notifi
 import { E } from '@endo/eventual-send';
 import { iterateLatest, makeFollower, makeLeader } from '@agoric/casting';
 
-import { unserializer } from 'utils/boardIdUnserializer';
+import { boardIdUnserializer } from 'utils/boardIdUnserializer';
```

Comment:
does this work? it doesn't need to be `'./utils/...`?

Reply:
Yes, webpack looks for it under `src` by default because of https://github.com/Agoric/dapp-amm/blob/main/ui/jsconfig.json